src/gc/gc.h

```diff
@@ -105,6 +106,7 @@ extern "C" uint8_t* g_gc_highest_address;
 extern "C" GCHeapType g_gc_heap_type;
 extern "C" uint32_t g_max_generation;
 extern "C" MethodTable* g_gc_pFreeObjectMethodTable;
+extern "C" MethodTable** g_gc_pStringClass;
```

Comment:
That lame to have nested pointer, but gc is initialized before EE sets pointers to method tables for such types.

Reply:
perhaps we should set it lazily, then?